docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -0,0 +1,105 @@
+
+= Time Series Service
+
+The time series service provides a scalable cloud-hosted time series database to store and persist performance metrics that {page-component-title} collects from the devices it monitors.
+
+In addition, users can view time series summary data including health status, capacity, and metrics per second in the OpenNMS Portal.
+
+== Requirements
+
+You must have the following to use the time series service:
+
+* Horizon 31+ (Meridian availability with the 2023 release)
+* Time series service subscription that provides access to the https://portal.opennms.com[OpenNMS Portal] to retrieve your activation key and view time series service metrics
+
+Contact sales@opennms.com to sign up for the time series service.
+
+== Deployment
+
+There are three phases of deployment to set up the time series service for your system:
+
+* Update OpenNMS Cloud Plugin files
+* Obtain an activation key for the service
+* Enable cloud services connectivity
+
+=== Update OpenNMS Cloud Plugin files
+
+The OpenNMS Cloud Plugin makes it possible for a {page-component-title} instance to send time series data to an OpenNMS-hosted cloud service.
+
+This plugin is installed by default in {page-component-title} 31.
+However, you need to move the configuration files into the correct location and restart {page-component-title} as part of the time series service set up:
+
+[source, console]
+----
+cp /opt/opennms/etc/examples/opennms.properties.d/timeseries.properties /opt/opennms/etc/opennms.properties.d/timeseries.properties
+
+systemctl restart openmns
+----
+
+=== Obtain an activation key for time series services
+
+The activation key lets {page-component-title} connect with OpenNMS cloud services (currently the time series service).
+
+You must log in to the OpenNMS Portal to get this activation key, copy it, and then paste it into the appropriate screen in {page-component-title}.
+
+When you signed up for the time series service, you received an activation email from OpenNMS with a login for the OpenNMS Portal.
+
+To obtain the activation key, follow these steps:
+
+. If you have not already done so, activate your OpenNMS Portal account by clicking the *Account Activation* button in your welcome email.
+. On the Create Your OpenNMS Account screen, type a password in the *Password* box.
+. Click *Create My Account*.
+. Sign in to the OpenNMS Portal with your username and password.
+. Choose *OpenNMS Core Instances* in the left menu and click *Add New*.
+. Type in the name of your instance and click *Generate Activation Key*.
+. Copy the instance key and click *Save & Close*.
+
++
+NOTE: The key is valid for one hour.
+If you do not use it within that time, you must log back into the OpenNMS Portal and copy it from your instance.
```

Comment:
Users cannot renew/generate another one-time code (such functionality does not exist).

Workaround: they'll have to delete the old instance and create another.